Sources/API/V1/yorkie/v1/yorkie.proto

```
@@ -122,59 +120,59 @@ message PushPullChangesResponse {
 
-message RefreshPresenceRequest {
+message WatchChannelRequest {
   string client_id = 1;
-  string presence_id = 2;
+  string channel_key = 2;
+}
+
+message WatchChannelResponse {
+  oneof body {
+    WatchChannelInitialized initialized = 1;
+    ChannelEvent event = 2;
```

Buf (1.69.0)

[error] cannot find `ChannelEvent` in this scope

(COMPILE)